### PR TITLE
Fix JSON Wire null double breaking parse

### LINKED ISSUE #953: JSONWire parser is confused when reading a field after null resolves to Double NaN
df307774 defines that NaN is turned into `null` on writing, and `null` is turned into NaN double on reading.
However, it seems that parsing such `null` sets parser in some bad state where it can no longer parse next fields in the same object.

I will attach a reproducer test.

### PR DESCRIPTION
**Pull request by alamar**
Fix general case of double parsing not skipping commas
closes #953

### CODE REVIEW

**Comment by alamar** [comment]
https://teamcity.chronicle.software/buildConfiguration/Chronicle_BuildAll_BumpedBranch/1082115?hideTestsFromDependencies=false&hideProblemsFromDependencies=false&expandBuildDeploymentsSection=false&expandBuildChangesSection=true all tests pass
@tgd please review

File: src/main/java/net/openhft/chronicle/wire/TextWire.java [hunk 1]
```diff
@@ -1932,7 +1932,7 @@ public <T> WireIn uint16(@NotNull T t, @NotNull ObjIntConsumer<T> ti) {
         @Override
         public <T> WireIn int32(@NotNull T t, @NotNull ObjIntConsumer<T> ti) {
             consumePadding();
-            ti.accept(t, (int) getALong());
+            ti.accept(t, (int) int64());
```
Comment (tgd) [inline src/main/java/net/openhft/chronicle/wire/TextWire.java:1935]: Why does this accept int64 when the calling method is called int32?

(I see this is not a new issue, it used to be a long before)

File: src/main/java/net/openhft/chronicle/wire/TextWire.java [hunk 2]
```diff
@@ -2743,6 +2734,7 @@ public long int64() {
 
             long l = getALong();
             checkRewind();
+            consumePadding(1);
```
Comment (tgd) [inline src/main/java/net/openhft/chronicle/wire/TextWire.java:2737]: What is the significance of this call to `consumePadding(1)`?

File: src/main/java/net/openhft/chronicle/wire/TextWire.java (hunk 1 above)
Comment (alamar) [reply src/main/java/net/openhft/chronicle/wire/TextWire.java:1935]: I really have no answer but I can add "enforce or throw" check here. I'm not advising to backport this change anyway and it seems safe to do in `ea`. WDYT?

File: src/main/java/net/openhft/chronicle/wire/TextWire.java (hunk 2 above)
Comment (alamar) [reply src/main/java/net/openhft/chronicle/wire/TextWire.java:2737]: That's the whole point of this change. When a comma supercedes a number, it gets consumed for free, but if there's whitespace before a comma (permitted by all specs) we need to explicitly consume this single comma. It is really "up to one".

**Review by tgd — APPROVED** [review]

**Pull request merged**